vcd/resource_vcd_rde.go

```diff
@@ -320,6 +339,16 @@ func resourceVcdRdeUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 		}
 	}
 
+	// Metadata is only supported since v37.0
+	if vcdClient.Client.APIVCDMaxVersionIs(">= 37.0") {
+		err = createOrUpdateOpenApiMetadataEntryInVcd(d, rde)
+		if err != nil {
+			return diag.Errorf("could not create metadata for the Runtime Defined Entity: %s", err)
+		}
+	} else if _, ok := d.GetOk("metadata_entry"); ok {
+		return diag.Errorf("metadata_entry is only supported since VCD 10.4.0")
+	}
+
```

Comment:
I am thinking now - we don't even support versions < 10.4.0 anymore with next version - maybe we can simplify it.
Comment works for other places and tests as well

Reply:
Done, thanks for the heads up.